genisys/make_password.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+from passlib.hash import pbkdf2_sha256
+from genisys.config_parser import YAMLParser
+
+class MakePassword:
+    """ Accepts command line argument to generate user password """
+    def __init__(self, config: YAMLParser):
+        self.config = {}
+        self.config["users"] = config.get_section("Users")
+    # end __init__
+     
+    def make_password(self, raw_password, filename):
+        """ Encrypts user password in hashed format and ansible vault format """
+        # use Passlib library to encrypt raw_ password, in /etc/shadow format in the "root-password" key under the "Users" section of the genisys config.
+        secret_password = pbkdf2_sha256.hash(raw_password)
+        self.config["users"]["root-login"] = secret_password
```

Comment:
You can probably drop this line. It doesn't look like it's getting used later due to the changes to how you're accessing the config file.